lib/pharos/logging.rb

```diff
@@ -17,7 +17,11 @@ def self.format_exception(exc, severity = "ERROR")
         backtrace = "\n    #{exc.backtrace.join("\n    ")}"
       end
 
-      "Error: #{exc.message.strip}#{backtrace}"
+      if exc.is_a?(Excon::Errors::CertificateError)
+        "Error: #{exc.message.lines.first[/(.+?\))/, 1]}#{backtrace}"
```

Comment:
Is this really safe? Does excon always have `(something)` in place that we can use for search?